tests/console/validate_multipath.pm

```diff
@@ -0,0 +1,172 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate multipathing on the installed system.
+# The multipathing support in SUSE Linux Enterprise Server is based on
+# the Device Mapper Multipath module of the Linux kernel and the multipath-tools user space package.
+# the Multiple Devices Administration utility (multipath) can be used to view the status of multipathed devices.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub verify_packages_installed {
+    my ($pkg_ref) = shift;
+    record_info('pkgs', 'Verify required packages installed');
+    assert_script_run("rpm -qa | grep $_",
+        fail_message => "$_ is not installed in the system") for (@{$pkg_ref});
+}
+
+sub verify_kernel_modules_loaded {
+    my ($mod_ref) = shift;
+    record_info('kernel module', 'Verify required kernel modules loaded');
+    assert_script_run("lsmod | grep ^$_",
+        fail_message => "$_ kernel module is not loaded") for (@{$mod_ref});
+}
+
+sub verify_services_running {
+    my ($srv_ref) = shift;
+    record_info('services', 'Verify required systemd services running');
+    validate_script_output("systemctl show -p SubState --value $_", qr/running/) for (@{$srv_ref});
+}
+
+sub verify_worldwide_id_used {
+    my ($wwid) = shift;
```

Comment:
because it is is written world-wide identifier in the docs.

Reply:
this is completely ambiquous. Even if the documentation uses it, it still abbreviation not a word

Reply:
what I mean is that because it could be a dash '-' in the name I can choose `verify_worldwide_id_used` or `verify_world_wide_id_used` due to `world-wide` could be one word.
is it clear now?